a3p-integration/proposals/z:acceptance/test-lib/sync-tools.js

```diff
@@ -0,0 +1,285 @@
+/**
+ * @file The purpose of this file is to bring together a set of tool that
+ * developers can use to synchronize operations they carry out in their tests.
+ *
```

Comment:
If this is a stand-in for the code in #10171, please say so, and indicate that it will be dropped in favor of importing those when they're ready.

Reply:
Resolved in https://github.com/Agoric/agoric-sdk/pull/10229/commits/50a0e17318c083daea6fc2628216adf294525e7a